articles/logic-apps/logic-apps-gateway-connection.md

```diff
@@ -101,7 +101,7 @@ After you create your gateway resource and associate your Azure subscription wit
 
 1. In the Azure portal, create or open your logic app in the Logic App Designer.
 
-1. Add a connector that supports on-premises connections, for example, **SQL Server**.
+1. Add a connector that supports on-premises connections, for example, **SQL Server**. Specifcally, this needs to be a [managed connector](/azure/connectors/managed#on-premises-connectors), not a [built-in connector](/azure/connectors/built-in).
```

Comment:
@eugman, thanks for your suggestion! Here's my proposed edit. If you agree, please manually accept the edit. Thanks!

```suggestion
1. Add a connector that supports on-premises connections. If this connector has both a [managed version](../connectors/managed.md#on-premises-connectors) and a [built-in version](../connectors/built-in.md), make sure that you use the managed version.
```

Reply:
Hi thanks for the reminder, I'm still a bit new at this pull request thing :).